sessions/js-forms-2/simple-todolist/README.md

```diff
@@ -0,0 +1,27 @@
+# Simple To-do-List
+
+Everyone has something to do. Let's make sure that no one forgets anything.
+
+## Task
+
+### 1. Listen to the submit event
+
+- Write an event listener for the form submit.
+- Log the user data to the console (we will use the data later in this exercise).
+- After submit, reset the form and focus the input field with the name `headline`.
+
+### 2. Create list entries
+
+Look at the index.html file: there already is an `ul` for our submitted todos. We only need to write a function which creates list items and appends them to this list.
+
+Here are some hints to guide you:
+
+- The function needs one parameter (which will be the form data object).
+- The list item
+  - has a headline and a text element ("task").
+  - `textContent` of headline and task are set via the parameter (you can use dot notation: `data.headline` and `data.task`).
```

Comment:
This is one possible way of solving the task, maybe we leave this open for the students to decide if they want to create each item themselves or use `innerHTML`?

Reply:
Do we really want them to use `innerHTML`? I see your point that this is one way to create a quick solution, but I am not sure whether we should propagate this option.

Do you have an idea how we rephrase the hint in a way that leaves it open, but does not use `innerHTML` automatically?

Reply:
Hmm, not sure about innerHTML here. If we want them not to use it we sould highlight that:
- solve this task without the use of innerHTML since it is unsafe when using it with user input strings.